### DISCUSSION TITLE
Check if an object has 'broken' warnings, information

### DISCUSSION THREAD

**Discussion by Chicagoan2016** [MarimerLLC/csla#3413] [Questions] [answered]
It appears 'IsValid' and 'IsSelfValid' checks for rules with severity of Error.
How could we check if an object has 'broken' warnings or information?

Kind Regards

**Comment by Chicagoan2016** [comment]
@rockfordlhotka  could we add this as an enhancement in future releases?

**Comment by rockfordlhotka** [comment, marked as answer]
The intent behind the CSLA design in this case is that you override `SaveAsync` to implement your desired behavior.

Often this means basically doing a copy-paste of the CSLA `SaveAsync` implementation and then altering that code to meet your needs.

This isn't a commonly requested thing, so I don't know that I'll take the time to change the core framework behavior. It is the sort of thing you can do in a custom base class, for example.
